<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>598</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <kerning>false</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(75, 25, 25);</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <widget class="QWidget" name="page">
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>470</y>
          <width>161</width>
          <height>61</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
border: 2px solid black;
border-radius: 9px;
color: white;
}
QPushButton:hover {
    background: rgba(99, 7, 7, 0.8)
}
QPushButton:pressed {
    background: rgba(99, 7, 7, 0.8)
}</string>
        </property>
        <property name="text">
         <string>Importar actualizacion vinos</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>-30</x>
          <y>-20</y>
          <width>861</width>
          <height>474</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/logo/logo.jpg&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_2">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayoutPage2">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinimumSize</enum>
        </property>
        <item>
         <spacer name="verticalSpacerTop">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">
            color: rgb(86, 7, 12);
            background-color: rgb(255, 255, 240);
            font-family: &quot;montserrat&quot;, Times, serif;
            font-size: 16px;
            font-weight: bold;
            border-radius: 5px;
            padding: 10px;
           </string>
          </property>
          <property name="text">
           <string>Seleccione las bodegas que desea actualizar</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="tableWidget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>50</horstretch>
            <verstretch>50</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">
            background-color: rgb(69, 5, 9);
            color: rgb(255, 255, 240);
            QHeaderView::section {
              color: rgb(255, 255, 240);
              background-color: rgb(255, 255, 240);
            }
            QTableWidgetItem::setData(Qt::BackgroundRole,Qt::grey)
           </string>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAsNeeded</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacerBottom">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonVolverBodega">
          <property name="styleSheet">
           <string notr="true">QPushButton {
border: 2px solid black;
border-radius: 9px;
color: white;
}
QPushButton:hover {
    background: rgba(99, 7, 7, 0.8)
}
QPushButton:pressed {
    background: rgba(99, 7, 7, 0.8)
}</string>
          </property>
          <property name="text">
           <string>Volver al inicio</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_3">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayoutPage3">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinimumSize</enum>
        </property>
        <item>
         <spacer name="verticalSpacerTop">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">
            color: rgb(86, 7, 12);
            background-color: rgb(255, 255, 240);
            font-family: &quot;montserrat&quot;, Times, serif;
            font-size: 16px;
            font-weight: bold;
            border-radius: 5px;
            padding: 10px;
           </string>
          </property>
          <property name="text">
           <string>Vinos actualizados</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="tableVinoActualizados">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>50</horstretch>
            <verstretch>50</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">
            background-color: rgb(69, 5, 9);
            color: rgb(255, 255, 240);
            QHeaderView::section {
              color: rgb(255, 255, 240);
              background-color: rgb(255, 255, 240);
            }
            QTableWidgetItem::setData(Qt::BackgroundRole,Qt::grey)
           </string>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAsNeeded</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">
            color: rgb(86, 7, 12);
            background-color: rgb(255, 255, 240);
            font-family: &quot;montserrat&quot;, Times, serif;
            font-size: 16px;
            font-weight: bold;
            border-radius: 5px;
            padding: 10px;
           </string>
          </property>
          <property name="text">
           <string>Vinos creados</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="tableVinoCreados">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>50</horstretch>
            <verstretch>50</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">
            background-color: rgb(69, 5, 9);
            color: rgb(255, 255, 240);
            QHeaderView::section {
              color: rgb(255, 255, 240);
              background-color: rgb(255, 255, 240);
            }
            QTableWidgetItem::setData(Qt::BackgroundRole,Qt::grey)
           </string>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAsNeeded</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonVolver">
          <property name="styleSheet">
           <string notr="true">QPushButton {
border: 2px solid black;
border-radius: 9px;
color: white;
}
QPushButton:hover {
    background: rgba(99, 7, 7, 0.8)
}
QPushButton:pressed {
    background: rgba(99, 7, 7, 0.8)
}</string>
          </property>
          <property name="text">
           <string>Volver al inicio</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonFinalizar">
          <property name="styleSheet">
           <string notr="true">QPushButton {
border: 2px solid black;
border-radius: 9px;
color: white;
}
QPushButton:hover {
    background: rgba(99, 7, 7, 0.8)
}
QPushButton:pressed {
    background: rgba(99, 7, 7, 0.8)
}</string>
          </property>
          <property name="text">
           <string>Finalizar</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacerBottom">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="recursos.qrc"/>
 </resources>
 <connections/>
</ui>